src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetForm/WidgetForm.styles.ts

```diff
@@ -23,6 +24,7 @@ export const getWidgetFormStyles = memoizeFunction((theme: Theme) => {
         ],
         widgetFormContents: [
             classNames.widgetFormContents,
+            leftPanelBuilderBlock,
```

Comment:
LOVE IT!! Could we make a common style for the stacks taht I commented on and just drop this in there? That might be even better than putting the class name on each stack directly.